portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function expand_collapsible(element) {
+  var content = element.nextElementSibling;
```

Comment:
[use `const`, not `var`](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let)